client/components/LearnerViewDirective.js

```diff
@@ -584,12 +581,7 @@ tie.directive('learnerView', [function() {
             var syntaxErrorIndex = feedback.getSyntaxErrorIndex();
```

Comment:
Do we even need the concept of "syntaxErrorIndex" any more? If not, would it be worth trying to remove it from the entire codebase?